README.md

```diff
@@ -2,17 +2,17 @@
 
 This is an implementation of Wayland McWayface [Part 3](https://drewdevault.com/2018/02/28/Writing-a-wayland-compositor-part-3.html) by Drew DeVault. 
 In order to have bindings of wlroots and wayland within java, I've made use of the jextract utility provided by project panama. 
-However, I've also made use of the `foreign` api directly as well, which can be seen in `usr.include.stdlib` in the scala sources.
+I've also made use of the `foreign` api directly which can be seen in `usr.include.stdlib` in the scala sources.
 
 ## Dependencies
 
-In order to build and run this project you will need [build ea+44 of project panama on jdk 13](https://jdk.java.net/panama/). 
-You'll also need the wayland development libraries, the wlroots library and development libraries, clang, libpixman's development libraries, libxkbcommon and possibly others. 
+To build and run this project, you will require [build ea+44 of project panama on jdk 13](https://jdk.java.net/panama/).
+The following will be required as well: the wayland development libraries, the wlroots library and development libraries, clang, libpixman's development libraries, libxkbcommon and possibly others. 
 
 ## Building
 
-To build this project, you can run `sbt compile` in the root of the project directory. If the build fails, please check to make sure that these settings in build.sbt
-
+To build this project, run `sbt compile` in the root of the project directory. 
+If the build fails, please check to make sure that the following settings are in build.sbt
```

Comment:
The settings will be in build.sbt, the issue is that they may not be set to the right paths. Please change the wording to "the following settings in build.sbt" to allow this line to flow into the bottom "point to your xdg-shell-unstable-v6.xml..."  line.